go.mod

```diff
@@ -76,6 +79,9 @@ replace (
 	k8s.io/helm => k8s.io/helm v2.13.1+incompatible
 	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.18.10
 	k8s.io/kube-openapi => github.com/gardener/kube-openapi v0.0.0-20200831104310-b5db060350c7
+	k8s.io/kube-scheduler/v18 => ./third_party/kube-scheduler/v18 // update those modules as well when updating kubernetes version
+	k8s.io/kube-scheduler/v19 => ./third_party/kube-scheduler/v19 // update those modules as well when updating kubernetes version
+	k8s.io/kube-scheduler/v20 => ./third_party/kube-scheduler/v20 // update those modules as well when updating kubernetes version
```

Comment:
I tried to vendor this change in provider-aws and I would say it is impossible. 
May be I am not educated enough, but I failed to vendor newer version of this repo trying with different `replace`s and `exclude`s - nothing works.